src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -20,6 +22,7 @@ const testWorkspace: WorkspaceAttribute = {
 describe('workspace service', () => {
```

Comment:
could we elaborate a little more for test description

```suggestion
describe('workspace service api integration test', () => {
```